build-tools/packages/build-tools/src/fluidBuild/tasks/leaf/leafTask.ts

```diff
@@ -738,13 +738,10 @@ export abstract class LeafWithGlobInputOutputDoneFileTask extends LeafWithFileSt
 		const globs = mode === "input" ? await this.getInputGlobs() : await this.getOutputGlobs();
 		const excludeGitIgnoredFiles: boolean = this.gitIgnore.includes(mode);
 
-		const files = await globby(globs, {
+		return globWithGitignore(globs, {
 			cwd: this.node.pkg.directory,
-			// file paths returned from getInputFiles and getOutputFiles should always be absolute
```

Comment:
Something similar to this comment feels worth moving to the docs of `globWithGitIgnore` to makes its expectations clear, probably the docs of its `patterns` parameter. And/or make the behavior controllable with `GlobWithGitIgnoreOptions`.